api-inference-community/docker_images/pyannote/requirements.txt

```diff
@@ -0,0 +1,4 @@
+starlette==0.14.2
+api-inference-community==0.0.19
+git+https://github.com/pyannote/pyannote-audio.git@develop#egg=pyannote-audio
+speechbrain == 0.5.10
```

Comment:
Is speechBrain needed? I think it can be removed

Reply:
It is necessary for `pyannote/speaker-diarization` to work.
However, I don't plan to make it a hard `pyannote-audio` dependency.

Reply:
Perfect to add it here then.